stdlib/universe/holt_winters/holt_winters.go

```diff
@@ -215,25 +198,25 @@ func (r *HoltWinters) forecast(params []float64, onlyFit bool) []float64 {
 	if onlyFit || r.includeFitData {
 		size += l
 	}
-	r.tracker.track(size)
-	fcast := make([]float64, 0, size)
+	fcast := mutable.NewFloat64Array(r.alloc)
```

Comment:
Because what `forecast` forecasts is used outside of its scope. The caller should release the new array. If `forecast` releases `fcast`, then it couldn't be used outside of its scope.